sw/src/service-worker/service-worker.js

```diff
@@ -0,0 +1,395 @@
+import { clientsClaim } from 'workbox-core'
```

Comment:
There's a lot in this file - reading [the docs I have so far](https://github.com/dhis2/app-platform/pull/550/files#diff-5d7ee5078b758edd2cca61ece335d6b75f3e0b4df95464e9ba1ee570e9635a90) should be a helpful guide. I'm working on adding more guiding comments today

Reply:
Could we split this up into multiple smaller files?

Reply:
Certainly